README.md

```diff
@@ -93,7 +96,7 @@ disponibles para backend o invocando directamente al API de Onepay.
 callback el comercio debe hacer el confirmación de la transacción, para lo cual dispone de 30 segundos desde que la
 transacción se autorizo, de lo contrario esta sera automáticamente reversada.
 
-    El callback sera invocado via `POST` e iran los parametros `occ` y `externalUniqueNumber` con los cuales podrás
+    El callback sera invocado via `POST` e irán los parametros `occ` y `externalUniqueNumber` con los cuales podrás
```

Comment:
será

Reply:
I thought that the callback was now going to be invoked using `GET`. Am I mistaken?

Reply:
You right, I am fixing that to the next push